test/e2e/kubectl.go

```diff
@@ -161,6 +165,7 @@ var _ = Describe("Kubectl client", func() {
 		It("should support exec", func() {
 			By("executing a command in the container")
 			execOutput := runKubectl("exec", fmt.Sprintf("--namespace=%v", ns), simplePodName, "echo", "running", "in", "container")
+
```

Comment:
I have found it to be more idiomatic to strike lines of this class in golang.  You don't have to do it though.